bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -223,8 +195,8 @@ def predict(self):
                     class_id = np.argmax(scores)
                     # Get confidence from score
                     confidence = scores[class_id]
-                    # Static threshold
-                    if confidence > 0.5:
+                    # First threshold to decrease candidate count and inscrease performance
```

Comment:
Why first?

Reply:
Because there is a second confidence barrier later on in the processing, but we lose like 4 FPS on my laptop if we carry all the zero confidence candidates all the way with us.